snippets/c_api/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: all indent clean
+
+ALOHALYTICS_ROOT=../../Alohalytics
+BUILD_DIR := build
+
+CPP=g++
+CFLAGS=-Wall -Werror -O3 -fpic
+CPPFLAGS=-std=c++11 $(CFLAGS) -I$(ALOHALYTICS_ROOT)/src -c
+
+SRC=$(wildcard *.cc)
+OBJ=$(SRC:%.cc=$(BUILD_DIR)/%.o)
+SO=$(OBJ:%.o=%.so)
+
+all: build ${SO}
```

Comment:
Please, don't mix in the same Makefile ```$(VAR)``` and ```${VAR}```. As far as I know, both notations are equivalent.